index

```diff
@@ -1,13 +0,0 @@
-#include<bits/stdc++.h>
```

Comment:
1) Naming Convention: Variable names t1, t2, and nextTerm should be more descriptive, such as term1, term2, and next_term.
2) Commenting: The code lacks proper comments explaining the logic and purpose of each section.
3) User Input Validation: The code does not validate if the user input is a positive number or not.
4) Code Formatting: The code should be properly indented and formatted for better readability.
5) Function Modularization: The Fibonacci series logic should be separated into a function for better code organization and reusability.